build-logic-commons/basics/src/main/kotlin/gradlebuild/basics/kotlindsl/kotlin-dsl-upstream-candidates.kt

```diff
@@ -1,12 +1,20 @@
+/**
+ * TODO: Remove once with Gradle 9.0, used so org.gradle.kotlin.dsl.* is kept
+ */
+@file:Suppress("UnusedImport")
+
 // This file contains members that we should consider to pull upstream
 // and make available to all Kotlin DSL users
 package gradlebuild.basics.kotlindsl
 
-import org.gradle.api.Project
-import org.gradle.kotlin.dsl.support.serviceOf
 import org.gradle.process.ExecOperations
 import java.io.ByteArrayOutputStream
 import java.io.File
+/**
+ * Used to import assign for Gradle 9.0
+ * TODO: Remove once with Gradle 9.0
+ */
+import org.gradle.kotlin.dsl.*
```

Comment:
❓ Can we be more specific in what we are importing here? Is it just `assign`?

Reply:
Let me try

Reply:
It seems it works, yes, thanks!